src/desktop/apps/order2/server.js

```diff
@@ -12,7 +12,7 @@ app.get('/order2/:orderID*', async (req, res, next) => {
   try {
     const user = req.user && req.user.toJSON()
 
-    const { ServerApp, redirect, status } = await buildServerApp({
+    const { ServerApp, redirect, status, headTags } = await buildServerApp({
```

Comment:
Will we need to do this same thing for the other new-style apps? (artist, collect)

Reply:
Not unless they needed to set the title via the app.